homeassistant/components/switchbot_cloud/entity.py

```diff
@@ -48,3 +49,17 @@ async def send_api_command(
             command_type,
             parameters,
         )
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        self.set_attributes()
+        super()._handle_coordinator_update()
+
+    def set_attributes(self) -> None:
```

Comment:
```suggestion
    def _set_attributes(self) -> None:
```
can we prefix it since it is only used internally by this class and subclasses?

Reply:
Updated 👍